corehq/apps/users/dbaccessors.py

```diff
@@ -94,14 +123,47 @@ def get_commcare_users_by_filters(domain, user_filters, count_only=False):
     """
     if not any([user_filters.get('role_id', None), user_filters.get('search_string', None),
                 user_filters.get('location_id', None), count_only]):
-        return get_all_commcare_users_by_domain(domain)
+        if user_type == MOBILE_USER_TYPE:
+            return get_all_commcare_users_by_domain(domain)
+        if user_type == WEB_USER_TYPE:
+            return get_all_web_users_by_domain(domain)
 
-    query = _get_es_query(domain, user_filters)
+    query = _get_es_query(domain, user_type, user_filters)
 
     if count_only:
         return query.count()
     user_ids = query.scroll_ids()
-    return map(CommCareUser.wrap, iter_docs(CommCareUser.get_db(), user_ids))
+    return map(CouchUser.wrap_correctly, iter_docs(CommCareUser.get_db(), user_ids))
```

Comment:
nit: Probably does not matter, but should this by `CouchUser.get_db()`?

Reply:
skipping this

Reply:
Fine to skip but just correcting on the typo i made,

instead of 
```
map(CouchUser.wrap_correctly, iter_docs(CommCareUser.get_db(), user_ids))
```
should this be
```
map(CouchUser.wrap_correctly, iter_docs(CouchUser.get_db(), user_ids))
```
since its returning couch users now and not commcare users.